bundles/AdminBundle/Controller/Admin/DataObject/DataObjectHelperController.php

```diff
@@ -2073,7 +2073,8 @@ protected function getCsvFieldData(Request $request, $field, $object, $requested
                 // check for objects bricks and localized fields
                 if (DataObject\Service::isHelperGridColumnConfig($field)) {
                     if ($helperDefinitions[$field]) {
-                        return DataObject\Service::calculateCellValue($object, $helperDefinitions, $field, ['language' => $requestedLanguage]);
+                        $val = DataObject\Service::calculateCellValue($object, $helperDefinitions, $field, ['language' => $requestedLanguage]);;
```

Comment:
not necessary to store it in a var and then return it.

Reply:
not really related :-)

Reply:
sure, you changed it with this PR